lib/registration.pm

```diff
@@ -265,7 +265,7 @@ sub fill_in_registration_data {
             assert_screen('scc-beta-filter-checkbox');
             send_key('alt-i');
         }
-        assert_screen($modules_needle);
+        send_key_until_needlematch $modules_needle, "down", 40;
```

Comment:
I'm really sorry, but this looks like another temporary solution, as there can be a gap between modules, so that they are not shown on the same screen, even if the needle was properly created and has each module as separate area in the needle. So we will have to touch this piece of code again, similarly like I had a hope that maximized window will solve this. What's about introducing mechanism which asserts each module separately in case `$modules_needle` didn't match or based on parameters?

Reply:
thanks for the quick feedback,that would be weak solution, you're right. I will try to propose some iteration.

Reply:
No rush, take your time. We should apply better solutions, which will give us space for innovations. If you don't find anything better, we can simply reconsider this test suite and rewrite it completely e.g. by activation module using SUSEConnect, validating that it's properly displayed in YaST, deactivating it again, etc.